src/client/store/explorer/selection/selectors/index.ts

```diff
@@ -27,21 +30,54 @@ const _baseSelectors = [
   (_state: RootState, _assessmentName: AssessmentName, cycleName: CycleName) => cycleName,
   (_state: RootState, _assessmentName: AssessmentName, _cycleName: CycleName, sectionName: SectionName) => sectionName,
 ]
-const getAxisSelection = createSelector(_baseSelectors, (filters, assessmentName, cycleName, sectionName) => {
-  return filters?.[assessmentName]?.[cycleName]?.axis?.[sectionName] ?? defaultAxisSelection
-})
 
-const getDimensions = createSelector(_baseSelectors, (filters, assessmentName, cycleName, sectionName) => {
-  return filters?.[assessmentName]?.[cycleName]?.dimensions?.[sectionName] ?? []
-})
+const getAxisSelection = createSelector(
+  _baseSelectors,
+  (
+    filters: ExplorerSelectionState | undefined,
+    assessmentName: AssessmentName,
+    cycleName: CycleName,
+    sectionName: SectionName
+  ) => {
+    return filters?.[assessmentName]?.[cycleName]?.axis?.[sectionName] ?? defaultAxisSelection
+  }
+)
+
+const getDimensions = createSelector(
+  _baseSelectors,
+  (
+    filters: ExplorerSelectionState | undefined,
+    assessmentName: AssessmentName,
+    cycleName: CycleName,
+    sectionName: SectionName
+  ) => {
+    return filters?.[assessmentName]?.[cycleName]?.dimensions?.[sectionName] ?? []
+  }
+)
 
-const getMeasures = createSelector(_baseSelectors, (filters, assessmentName, cycleName, sectionName) => {
-  return filters?.[assessmentName]?.[cycleName]?.measures?.[sectionName] ?? []
-})
+const getMeasures = createSelector(
+  _baseSelectors,
+  (
+    filters: ExplorerSelectionState | undefined,
+    assessmentName: AssessmentName,
+    cycleName: CycleName,
+    sectionName: SectionName
+  ) => {
+    return filters?.[assessmentName]?.[cycleName]?.measures?.[sectionName] ?? []
+  }
+)
 
-const getUnits = createSelector(_baseSelectors, (filters, assessmentName, cycleName, sectionName) => {
-  return filters?.[assessmentName]?.[cycleName]?.units?.[sectionName]
-})
+const getUnits = createSelector(
+  _baseSelectors,
+  (
+    filters: ExplorerSelectionState | undefined,
```

Comment:
Aren't the types inherited ? Why is it needed to specify them ? ( out of curiosity )